arches_references/templates/views/components/plugins/controlled-list-manager.htm

```diff
@@ -1 +1,16 @@
-<div id="controlled-list-manager-mounting-point"></div>
+<div class="css-reset">
+    <div
+        id="controlled-list-manager-mounting-point"
+        style="font-family: 'Open Sans'"
+    ></div>
+</div>
+
+<style>
+.css-reset * {
+    all: revert-layer;
+}
+
+.fa {
+    font-family: fontAwesome;
+}
+</style>
```

Comment:
There might be a way to hide some of this complexity in the PluginView. But on the other hand, we might want to keep this as explicit as possible for now, so we can copy-pasta it for future core arches micro-app Vue work.

Reply:
If possible, I'd like to leverage the need we have to build out lingo as a forcing function to get this in core. I'm unsure if `revert-layer` will handle the all the use cases we'd need it to. 

Arches already has a `reset.css` file, is it possible to leverage that in a classname that can get hung on any `createVueApplication` mounting point to affect it and all its children?

Reply:
Just adding a small note that I did test adding selectors in arches.scss e.g.:

```css
.actions {
    background-color: purple
}
```
and with this change they no longer appeared in the controlled list manager on the item rows where I use the class name "actions".